pkgs/by-name/ad/adminer/package.nix

```diff
@@ -40,21 +40,19 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
-  passthru = {
-    updateScript = nix-update-script { };
-  };
+  passthru.updateScript = nix-update-script { };
```

Comment:
I prefer the unpacked syntax since it makes diff clearer when adding more passthru attributes.